.bazelrc

```diff
@@ -71,17 +71,6 @@ build:windows --color=yes
 build:clang-cl --per_file_copt="-\\.(asm|S)$@-fansi-escape-codes"
 build:clang-cl --per_file_copt="-\\.(asm|S)$@-fcolor-diagnostics"
 
-build:manylinux2010 --copt="-Wno-unused-result"
-build:manylinux2010 --linkopt="-lrt"
-
-# Debug build flags. Uncomment in '-c dbg' builds to enable checks in the C++ standard library:
```

Comment:
I think we should keep those debug flags, I have found them very useful in the past when debugging some crazy STL corruptions :)

Reply:
is it possible to move these to some docs or something, rather than putting them in default `.bazelrc` in comment form?

Reply:
adding `*-debug` flavors for the debug flags.